laravel-app/database/seeders/UserInformationSeeder.php

```diff
@@ -29,9 +31,11 @@ public function run(): void
                 'middle_name' => 'Elanor',
                 'bio' => 'Hello World',
                 'gender' => 'male',
+                'created_at' => now(),
+                'updated_at' => now(),
             ],
         ];
 
-        DB::table('user_information')->insert($user_information);
+        UserInformation::insert($user_information);
```

Comment:
Is this working? AFAIK if you want to create data using eloquent it should be **create()** right?